crates/containerd-shim-wasmedge/src/instance.rs

```diff
@@ -54,7 +56,7 @@ impl Engine for WasmEdgeEngine {
             .register_module(Some(&mod_name), module)
             .context("registering module")?;
 
-        log::debug!("running with method {func:?}");
+        containerd_shim_wasm::debug!(ctx, "running with method {func:?}");
```

Comment:
is it possible to do something like: 

```suggestion
        containerd_shim_wasm::debug!(ctx, func=func; "running method");
```

so down stream users could add there own structured logs?

Reply:
I don't think this is supported. do you think we need to support it?

Reply:
No not necessarily, I though it might just work with the marco as is or small tweak. It is a nice to have but not critical, a downstream consumer could bypass our convenience macro and pass their own structure if they wanted too

Reply:
okay if there is a demand for it I will add more macros for it.